stable/artifactory-ha/templates/nginx-deployment.yaml

```diff
@@ -46,11 +60,36 @@ spec:
                 sleep 5; nginx -s reload; touch /var/log/nginx/conf.done
         env:
         - name: ART_BASE_URL
+          {{- if .Values.nginx.env.artUrl }}
+          value: {{ .Values.nginx.env.artUrl }}
+          {{- else }}
           value: 'http://{{ $serviceName }}:{{ $servicePort }}/artifactory'
+          {{- end }}
         - name: SSL
           value: "{{ .Values.nginx.env.ssl }}"
         ports:
-        - containerPort: 80
-        - containerPort: 443
+        - containerPort: {{ .Values.nginx.internalPortHttp }}
+        - containerPort: {{ .Values.nginx.internalPortHttps }}
+        volumeMounts:
+        - name: nginx-volume
+          mountPath: {{ .Values.nginx.persistence.mountPath | quote }}
+        {{- if .Values.nginx.tlsSecretName }}
+        - name: ssl-secret-volume
+          mountPath: "/var/opt/jfrog/nginx/ssl"
+        {{- end }}
         resources:
 {{ toYaml .Values.nginx.resources | indent 10 }}
+      volumes:
+      - name: nginx-volume
+      {{- if .Values.nginx.persistence.enabled }}
+        persistentVolumeClaim:
+          claimName: {{ .Values.artifactory.persistence.existingClaim | default (include "artifactory-ha.nginx.fullname" .) }}
```

Comment:
Change `artifactory.persistence.existingClaim` to `nginx.persistence.existingClaim`

Reply:
done